Clarify what happens when limits are violated (#113)
w3c/baggage: baggage/HTTP_HEADER_FORMAT.md

> 1. A platform MUST propagate all `list-member`s up to _at least_ 64 `list-member`s including any `list-member`s added by the platform.

Reviewer: ```suggestion
1. A platform MUST propagate all `list-member`s including any `list-member`s added by the platform whenever both of these conditions are met:

**Condition 1:** The resulting `baggage-string` contains 64 `list-member`s or less.
**Condition 2:** The resulting `baggage-string` is of size 8192 bytes or less. 

2. If the resulting `baggage-string` would contain more than 64 `list-member`s, a platform MAY drop certain `list-member`s at its own discretion until the resulting `baggage-string` contains 64 `list-member`s or less.

3. If the resulting `baggage-string` would contain more than 8192 bytes, a platform MAY drop certain `list-member`s at its own discretion until the resulting `baggage-string` is of size 8192 bytes or less.
```

Reviewer 2: I like this, but I would not separate 2 and 3, as it's a lot of nearly identical wording just to say that if C1 or C2 or both are violated then drop the items as you see fit.

> 2. A platform MUST propagate all `list-member`s including any `list-member`s added by the platform if the resulting `baggage-string` would be 8192 bytes or less. If the resulting `baggage-string` would be greater than 8192 bytes, some `list-member`s MAY be dropped until the resulting `baggage-string` is 8192 characters or less.

Reviewer: ```suggestion
```

> 3. If either (1) or (2) is violated, the platform MAY drop `list-member`s at its own discretion. If a platform does not propagate all `list-member`s, it is left to the implementer to decide which `list-member`s to propagate.

Reviewer: ```suggestion
```
